utils/src/bulk-insert-patients.ts

```diff
@@ -0,0 +1,142 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+
+import { MetriportMedicalApi, PatientCreate, USState } from "@metriport/api";
+import csv from "csv-parser";
+import fs from "fs";
+
+const apiKey = getEnvVarOrFail("API_KEY");
+const facilityId = getEnvVarOrFail("FACILITY_ID");
+const apiUrl = getEnvVarOrFail("API_URL");
+
+const metriportAPI = new MetriportMedicalApi(apiKey, {
+  baseAddress: apiUrl,
+});
+
+async function main() {
+  const results: PatientCreate[] = [];
+  const errors: Array<{ firstName: string; lastName: string; dob: string; message: string }> = [];
+
+  fs.createReadStream("../insert-patients.csv")
+    .pipe(csv())
+    .on("data", async data => {
+      const metriportPatient = mapCSVPatientToMetriportPatient(data);
+      results.push(metriportPatient);
+    })
+    .on("end", async () => {
+      for (const [i, patient] of results.entries()) {
+        try {
+          await sleep(3000);
+          const createdPatient = await metriportAPI.createPatient(patient, facilityId);
+          console.log(i, createdPatient);
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        } catch (error: any) {
+          errors.push({
+            firstName: patient.firstName,
+            lastName: patient.lastName,
+            dob: patient.dob,
+            message: error.message,
+          });
+        }
+      }
+      console.log(errors);
+    });
+}
+
+function sleep(ms: number) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+const mapCSVPatientToMetriportPatient = (csvPatient: {
+  firstname: string;
+  lastname: string;
+  patient_dob: string;
+  address1: string;
+  city: string;
+  state_name: string;
+  zipcode: string;
+  phone: string;
+  gender: string;
+}): PatientCreate => {
+  return {
+    firstName: csvPatient.firstname,
+    lastName: csvPatient.lastname,
+    dob: csvPatient.patient_dob,
+    genderAtBirth: csvPatient.gender === "Female" ? "F" : "M",
```

Comment:
Shouldn't we explicitly filter out patients without gender? This code will set to `M` if `undefined`, I mean.

Reply:
I added gender for all patients but this is a good point for future csv's. Ill update

Reply:
Ill actually need to update the schema as we make gender mandatory

Reply:
Gender actually cant be optional based off the CW spec https://specification.commonwellalliance.org/services/rest-api-reference